src/connector/postgres/conversion.rs

```diff
@@ -116,6 +116,13 @@ impl GetRow for PostgresRow {
                     }
                     None => Value::Real(None),
                 },
+                PostgresType::BYTEA => match row.try_get(i)? {
+                    Some(val) => {
+                        let val: &[u8] = val;
```

Comment:
Is this line needed? If it's for inference of `try_get`, maybe use turbofish there instead?

Reply:
yeah it's the `try_get` above: https://docs.rs/tokio-postgres/0.5.5/tokio_postgres/row/struct.Row.html#method.try_get

Reply:
It's that way to align with the other branches. I think the tokio-postgres doc show something like that as well. Agreed I'd prefer a turbofish.